### ISSUE TITLE
TypeError: Cannot call method 'join' of undefined

### ISSUE THREAD

**Issue by dzoba** [facebook/regenerator#179]
I just started getting this error after this commit: https://github.com/facebook/regenerator/commit/1efe26ae772409548098d91b92787f50c0d42851

If I use ```runtime.path, "utf-8"``` instead of ```path.join(__dirname, "runtime.js"), "utf-8"``` everything runs fine.

Here is the full error:

```
minions/node_modules/analytics-node/node_modules/superagent-proxy/node_modules/proxy-agent/node_modules/pac-proxy-agent/node_modules/pac-resolver/node_modules/regenerator/main.js:55
        path.join(__dirname, "runtime.js"), "utf-8"
             ^
TypeError: Cannot call method 'join' of undefined
    at Function.compile (/vagrant/src/minions/node_modules/analytics-node/node_modules/superagent-proxy/node_modules/proxy-agent/node_modules/pac-proxy-agent/node_modules/pac-resolver/node_modules/regenerator/main.js:55:14)
    at Object.<anonymous> (/vagrant/src/minions/node_modules/analytics-node/node_modules/superagent-proxy/node_modules/proxy-agent/node_modules/pac-proxy-agent/node_modules/pac-resolver/index.js:36:41)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/vagrant/src/minions/node_modules/analytics-node/node_modules/superagent-proxy/node_modules/proxy-agent/node_modules/pac-proxy-agent/index.js:35:19)
    at Module._compile (module.js:456:26)
```

**Comment by gvilarino** [comment]
:+1: having this same problem and unable to build

**Comment by slifszyc** [comment]
:+1: on this issue, same error

**Comment by fkling** [comment]
There is already a pull request for that: #178

**Comment by dzoba** [comment]
If we can get this PR merged in, everything will be groovy https://github.com/facebook/regenerator/pull/178

**Comment by benjamn** [comment]
Should be fixed now, thanks to @xaka's PR #178!
